README.md

```diff
@@ -131,6 +131,15 @@ The calendar will support selecting multiple dates. NgModel will be set as an ar
 ```html
 <div pickadate multiple></div>
 ```
+
+##### not blank
+
+The calendar will not allow user to remove the last remaining date.
+
+```html
+<div pickadate multiple not-blank></div>
```

Comment:
I think that the default behaviour should be `not-blank`, and if the user wants to allow a blank date, it should add the attribute `allow-blank-date` or something like that. what do you think?